ludwig/schema/metadata/trainer_metadata.py

```diff
@@ -24,6 +24,22 @@
                                      expected_impact=ExpectedImpact.HIGH,
                                      literature_references=None,
                                      internal_only=False),
+      'auto_batch_size_ceiling': ParameterMetadata(ui_display_name='Auto Batch Size Ceiling',
```

Comment:
Nice! Maybe call it `max_batch_size` and have it also apply to `increase_batch_size_on_plateau`? Then we can have the auto batch size process also set this and then remove `increase_batch_size_on_plateau_max`. What do you think?

Reply:
Sounds reasonable! Will do this.

Reply:
Following up on this: if we unify `auto_batch_size_ceiling` with `increase_batch_size_on_plateau_max`, what should the default value be? Right now the default for `increase_batch_size_on_plateau_max` is 512, which I don't believe we want as the default for `auto_batch_size_ceiling`. Tentatively, we set the default for the new parameter `max_batch_size` to `None`, and we simply expect the user to pass this value in if they are interested in using the `increase_batch_size_on_plateau` functionality. WDYT? cc @dantreiman